flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java

```diff
@@ -277,11 +277,12 @@ public static Configuration forReporter(Configuration configuration, String repo
      * JobManager of an operator.
      */
     public static final ConfigOption<String> SCOPE_NAMING_JM_OPERATOR =
-            key("metrics.scope.jm-operator")
+            key("metrics.scope.coordinator")
```

Comment:
Do we want to move ahead with this change? If so, we should also change the naming internally.

Reply:
+1. The question is - do we expect any other types of "JM operators" to ever exist. If we do, in my opinion, we should keep the scope and treat the whole `/coodinator-metrics` request code path as a convenience filter on top, not a substitution to the broader category.

Reply:
Yes. Totally agreed. I will keep the scope the same. Upon further investigation, this is also aligned with the direction of evolving this API: https://lists.apache.org/thread/hdvz57wmlqxdxgnw73tcgbcfyo6vk09h